projects/rocprofiler-sdk/source/lib/python/rocprofv3/avail.py

```diff
@@ -74,12 +75,14 @@ def __init__(
         counter_description,
         counter_dimensions,
         is_hw_constant,
+        spm_support,
```

Comment:
Should we make this something like (in case spm_support is False by default?
```suggestion
        spm_support=False,
```

Reply:
Why would we do that here?

Reply:
That way, the old PMC code doesn't need to pass the False, so you'll have unchanged code that implicitly ignores SPM. Just a matter of preference.

Reply:
I would like to have it as any other member that gets initialized when instantiated